#! /bin/sh /usr/share/dpatch/dpatch-run
## menu_smart-sharpen-redux.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: better menu location for smart-sharpen-redux

@DPATCH@
diff -urNad gimp-plugin-registry~/smart-sharpen-redux/Eg-SmartSharpen.scm gimp-plugin-registry/smart-sharpen-redux/Eg-SmartSharpen.scm
--- gimp-plugin-registry~/smart-sharpen-redux/Eg-SmartSharpen.scm	2007-11-19 23:56:39.000000000 +0100
+++ gimp-plugin-registry/smart-sharpen-redux/Eg-SmartSharpen.scm	2007-12-12 20:06:36.000000000 +0100
@@ -124,5 +124,5 @@
 )
 ;
 (script-fu-menu-register "script-fu-Eg-SmartSharpen"
-			 "<Image>/Filters/Eg")
+			 "<Image>/Filters/Enhance/Sharpen (Smart Redux)")
 ;
